src/apps/competitions/unpackers/v1.py

```diff
@@ -7,11 +7,17 @@
 class V15Unpacker(BaseUnpacker):
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
+
+        # Just in case docker image is blank (""), replace with default value
+        docker_image = self.competition_yaml.get('competition_docker_image', 'codalab/codalab-legacy:py3')
```

Comment:
maybe do:
```python
docker_image = self.competition_yaml.get('competition_docker_image')
if not docker_image:
    docker_image = "codalab/codalab-legacy:py3"
```
so that we replace _every_ falsy value?

Reply:
I'll switch to this style